assets/js/contact.js

```diff
@@ -0,0 +1,11 @@
+/**	CONTACT FORM
```

Comment:
Where is the API for the contact form? It needs to get filled in if the contact form should work as intended.